src/Chapter1.hs

```diff
@@ -416,7 +416,7 @@ task is to specify the type of this function.
 >>> squareSum 3 4
 49
 -}
-
+squareSum :: Num a => a -> a -> a
```

Comment:
We explain `Num` in the following chapters, so you are one step ahead 😸

In here `Int -> Int -> Int` would also work, as a more specific type 🙂

Reply:
Ok, got it, I am reading learnyouahaskell right now, so maybe I am a bit ahead,  but before that we looked at the type of `(+)` so I thought that I should specify type with Num constraint.